apps/widget-configurator/src/app/configurator/consts.ts

```diff
@@ -1,59 +1,58 @@
-import { TradeType, TokenList } from '@cowprotocol/widget-lib'
+import { TradeType } from '@cowprotocol/widget-lib'
+
+import { TokenListItem } from './types'
 
 export const TRADE_MODES = [TradeType.SWAP, TradeType.LIMIT, TradeType.ADVANCED]
 
 // Sourced from https://tokenlists.org/
-export const TOKEN_LISTS: TokenList[] = [
-  { name: 'CoW Protocol', url: 'https://files.cow.fi/tokens/CowSwap.json' },
-  { name: '1inch', url: 'https://tokens.1inch.eth.link' },
-  { name: 'Aave Token List', url: 'https://tokenlist.aave.eth.link' },
-  { name: 'Agora dataFi Tokens', url: 'https://datafi.theagora.eth.link' },
+export const DEFAULT_TOKEN_LISTS: TokenListItem[] = [
+  { url: 'https://files.cow.fi/tokens/CowSwap.json', enabled: true },
+  { url: 'https://tokens.coingecko.com/uniswap/all.json', enabled: true },
+  { url: 'https://tokens.1inch.eth.link', enabled: false },
+  { url: 'https://tokenlist.aave.eth.link', enabled: false },
+  { url: 'https://datafi.theagora.eth.link', enabled: false },
   {
-    name: 'BA ERC20 SEC Action',
     url: 'https://raw.githubusercontent.com/The-Blockchain-Association/sec-notice-list/master/ba-sec-list.json',
+    enabled: false,
```

Comment:
I'll raise this again.
I don't think this list should be here.
This is a block list used by Uniswap.
Why would we want to users enable specifically the blocked tokens to be traded?
Also, not even sure they would work in our infra.

Reply:
agree. 
yes, pls delete

Reply:
Thanks! Removed it